### PR TITLE
HACKLE-5405  refactor: remove emotion in favor of css

### PR DESCRIPTION
**Pull request by yjang-hackle**
## !LOTS OF WHITESPACE CHANGES!
make sure to turn off whitespace changes during review
Screenshot 2022-12-30 at 10 05 51 AM

visual regression testing succeeding

original
localhost_8000_

changed
localhost_8000_ (2)

### CODE REVIEW

File: compile-css.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const fs = require('fs');
+const sass = require('sass');
```
Comment (yjang-hackle) [inline compile-css.js:3]: using `sass` instead of `node-sass`, cuz `node-sass` be crazy

File: package.json [hunk 2]
```diff
@@ -32,19 +33,15 @@
 		"test:watch": "mocha --require ts-node/register --require enzyme.ts --watch-extensions ts,tsx --watch ./test/**"
 	},
 	"dependencies": {
+		"bem-cn": "^3.0.1",
 		"classnames": "^2.2.6",
-		"create-emotion": "^10.0.14",
 		"diff": "^4.0.1",
-		"emotion": "^10.0.14",
```
Comment (yjang-hackle) [inline package.json:38]: removing emotion and using bem convention instead

File: src/index.tsx [hunk 3]
```diff
@@ -1,204 +1,189 @@
...
+  ) => JSX.Element;
+  // Event handler for line number click.
+  onLineNumberClick?: (
+    lineId: string,
+    event: React.MouseEvent<HTMLTableCellElement>,
+  ) => void;
+  // Array of line ids to highlight lines.
+  highlightLines?: string[];
+  // Style overrides.
+  classes?: ReactDiffViewerStylesOverride;
```
Comment (yjang-hackle) [inline src/index.tsx:54]: removing `styles` prop in favor of `classes` prop to make it easy to style w/ `makeStyles` or `withStyles` in dashboard

File: package.json (hunk 2 above)
Comment (teddy-hackle) [reply package.json:38]: 오... bem-cn 처음 보네요

File: compile-css.js (hunk 1 above)
Comment (teddy-hackle) [reply compile-css.js:3]: scss compile 아예 빼고 css 로만 동작하게 하는건...?

**Review by peter-hackle — APPROVED** [review]

File: compile-css.js (hunk 1 above)
Comment (yjang-hackle) [reply compile-css.js:3]: light/dark theme 때문에 variable 들이 있어서요. css 도 variable 이 되긴 하지만 ie11 support 가... https://caniuse.com/css-variables

Comment (teddy-hackle) [reply compile-css.js:3]: 이런....

**Pull request merged**